### DISCUSSION TITLE
Fulton County Mud Road Landfill Gas Capture (VCS 834)

### DISCUSSION THREAD

**Discussion by hai-yr** [opentaps/open-climate-investing#34] [Offsets]
Creating a thread to investigate the Fulton County Mud Road Landfill Gas Capture project in Johnstown, New York and its associated carbon credits. The project details can be seen here: https://registry.verra.org/app/projectDetail/VCS/834

**Comment by hai-yr** [comment]
Project Summary:

"The Fulton County Department of Solid Waste (FCDSW) owns and operates a landfill gas collection and control
system at the Mud Road Sanitary Landfill located in the Town of Johnstown, New York. The project activity
consists of the installation of an active gas collection and control system which collects landfill gas and sends it
to a landfill gas to energy facility for destruction. The destruction of methane in LFG that otherwise would be
vented to the atmosphere results in a net reduction in CO2-equivalents."

Other points:
- Project start date of June 4, 2010
- The project verification applies the CDM baseline and monitoring methodology, ACM0001
- The first crediting period is June 4, 2010 to June 3, 2020
- The estimated emission reductions during the first crediting period are 365,720 mt CO2e
- No credits were sought for displaced grid emissions for the power generated by the LFGTE facility.
- The project technology consists of the collection system (landfill gas extraction wells, horizontal collection piping, air operated dewatering pumps) and two Caterpillar G3520C internal combustion engine-generator sets, including associated monitoring equipment. Prior to the project, LFG was collected and destroyed in an open flare.
- Baseline emissions are calculated as the total quantity of methane destroyed by the Project (MDproject,)
minus the quantity of methane that would have been destroyed in the baseline scenario in the year 2006
(MDBL,2006), multiplied by the global warming potential for methane.

**Comment by hai-yr** [comment]
Initial Research Questions on Project:

- How might recent/upcoming legislation on methane affect the additionality of projects like this in the future?
- What GWP (global warming potential) was used for methane on the project?
[Edit: Global Warming Potential value for methane for the first commitment period is 21
tCO2e / tCH4]
- How long did it take to return the initial investment and is running the plant now profitable without the credits?

Potentially applicable known issues with landfill gas capture:
https://ec.europa.eu/clima/system/files/2017-04/clean_dev_mechanism_en.pdf

- Default assumptions for the rate of methane captured under pre-existing collection systems or regulations are unjustified and have the potential to overestimate emission reductions  “CDM landfill methane projects, for example, realize just 35% of their projected emissions reductions.” https://www.offsetguide.org/avoiding-low-quality-offsets/vetting-offset-projects/methane-capture/
https://www.scsengineers.com/wp-content/uploads/2015/03/Stege_Methane_Emission_Reductions_LFG_Projects_Developing_Countries.pdf

- Potential for perverse incentives for policy makers not to regulate landfills or enforcing regulations in place  

- Perverse incentives for project developers to manage landfills in ways that increase methane generation 

- Perverse incentives for policy makers not to pursue less GHG-intensive waste treatment methods, such as composting or incineration  

- Some landfill gas projects exclude waste pickers and informal sector recycling, reducing overall rates of reuse and recycling

**Reply by sichen1234** [reply]
The one I'm most concerned about is “CDM landfill methane projects, for example, realize just 35% of their projected emissions reductions."  How is that happening?  Does this project address that issue adequately?

Of the others, I feel:

1. "Potential for perverse incentives for policy makers not to regulate landfills or enforcing regulations in place" and "Perverse incentives for policy makers not to pursue less GHG-intensive waste treatment methods, such as composting or incineration"  --> This is a common concern.  For example, is it better to recycle more plastic or use less plastic?
2. Perverse incentives for project developers to manage landfills in ways that increase methane generation --> This could happen.  For example, carbon credits to reduce HFC-23 refrigerant gas caused more HFC-22 gas production.  But in reality, but how easy is that to do, and is it worth it for the developer to do this?
3. Some landfill gas projects exclude waste pickers and informal sector recycling, reducing overall rates of reuse and recycling.  --> Has this actually happened?  How do we know if it is happening specifically at this project?

**Reply by hai-yr** [reply]
<img width="309" alt="Screen Shot 2022-07-29 at 1 55 50 PM" src="https://user-images.githubusercontent.com/39865976/181841821-3e21a8d9-cde2-4394-976b-f29d8ed8e4da.png">
I am looking at this paper 
https://www.scsengineers.com/wp-content/uploads/2015/03/Stege_Methane_Emission_Reductions_LFG_Projects_Developing_Countries.pdf

1. Right, I just saw some news about how much investment is going into these projects: https://www.bizjournals.com/houston/news/2022/04/28/waste-management-rng-landfill-gas-investments.html#:~:text=Classified%20%26%20Legals-,Waste%20Management%20investing%20%24825M%20to%20add,natural%20gas%20plants%20at%20landfills&text=Jim%20Fish%2C%20president%20and%20CEO%20of%20Waste%20Management.&text=The%20investments%20will%20help%20WM,fleet%20with%20RNG%20by%202026.
This is also an interesting article, because it highlights that there are also business interests in separating food waste for more efficient conversion to gases.
https://www.wastedive.com/news/vanguard-renewables-neil-smith-blackrock-food-waste-rng/627871/

2. That is a good point... I will need to look into it more. 
3. I have a feeling there are less waste pickers in the US in New York than in developing countries. The only way to know for sure would likely be if it were in the project docs or in contacting someone at the project directly.

**Reply by hai-yr** [reply]
"The project has elected to use the US EPA LandGEM Landfill Gas Emissions model instead of the CDM Tool... This is more accurate for projecting landfill emissions in the future for landfills in the United States and was used in the ex ante emissions reductions analysis."

<img width="347" alt="Screen Shot 2022-08-05 at 5 59 36 PM" src="https://user-images.githubusercontent.com/39865976/183227498-44f567a5-41e5-4d30-948d-5266e42f9a75.png">

**Comment by sichen1234** [comment]
This is pretty interesting.  Does this mean that the equipment will capture the methane at a landfill and then combust it, instead of letting it leak into the atmosphere?  Is the calculation the difference of combusted versus leaked methane?  This should be in the original project description as well as the verification reports.

Yes, that GWP factor is too low now.  Even the 100 year GWP is now 29, and 20 year GWP is over 80x.  So that's very good for these credits.

On the other hand, how is it really monitored?  You can find this in the verification reports.  Are they reviewing the data?  Do they address the potential issues with the project?

Finally where can you buy these credits, and what vintages are available?

There's also the interesting question of buying very old vintage years -- are they just as effective as buying current year offsets?

**Comment by hai-yr** [comment]
"Immediately prior to the project being installed, the landfill gas collection and destruction system included a central candle stick flare unit, a Hauck blower, and extensive monitoring equipment." So before, the methane was solely flared through the central candle (maybe also leaked it separately?).

"The project achieves greenhouse gas emissions reductions through the collection and destruction of methane from the landfill, which converts the methane to carbon dioxide and prevents the methane from being vented to the atmosphere." The project generates electricity which is connected to the grid but not factored into the credits calculations.

"Because the landfill and the methane it generates would exist with or without the project and the project itself generates no new greenhouse gas emissions, the project was clearly not implemented to generate GHG emissions for the purpose of their subsequent reduction, removal or destruction." I think this is valid, but it does raise questions about future projects that have come after carbon markets have grown, and how the incentives could change.

<img width="864" alt="Screen Shot 2022-07-28 at 4 59 20 PM" src="https://user-images.githubusercontent.com/39865976/181656348-6bdf5ded-c8e5-420f-96ba-45a92409e198.png">

Flow meters and a Siemens analyzer monitor the quantity of landfill gas and the fraction of methane that is sent to the engines that run on the gas. Measurements from the SCADA system (connected to the engines) and manual operator logs when plant is operational. Plant operations from the first 6 months implied the plant is running 98.9% of the time. 

The credits are available on Patch. The vintage available is from 2020.

**Comment by sichen1234** [comment]
Interesting.  So a few questions based on this:

1. Does this mean that before the project, methane was flared with the central candle stick flare, and afterwards, methane is burnt to generate electricity and fed back to the grid?
2. How is the amount of methane destroyed/combusted in absence of project obtained?
3. Where do they get the CO2 emissions intensity of baseline electricity displaced?
4. How do they justify that this project is "additional"?

Also, does any third party research on landfill gas capture point to potential issues about these measurements?

**Reply by hai-yr** [reply]
Yes, some of the methane is burnt to generate energy and some is sent to natural gas distribution networks.

The amount flared can be determined through this formula: MDflared,2006 = (LFGflare,2006 * wCH4,2006 * DCH4). They had some flow measurements and also measurements of the fraction of the gas that was methane.
"It is assumed that all landfill gas sent to the flare was combusted in the pre-project scenario. This is very conservative because CDM allows a default combustion efficiency of 0.9 for open flares without continuous monitoring which would reduce the amount of landfill gas destroyed in the pre-project scenario." 

So the electricity produced isn't factored into the emissions reduction. The emissions saved are essentially the amount of methane that is destroyed by the project vs. what would be destroyed in the baseline:
BEy = ( MDproject,y – MDBL,2006) * GWPCH4

Additionality - so this is one I'm starting to pick up some red flags as it seems that the longer the project goes on,  the more that the project has been paid back -- I'd be interested to see what the operating costs vs. revenues are now 14 years after the project began.
<img width="850" alt="Screen Shot 2022-07-29 at 12 05 24 PM" src="https://user-images.githubusercontent.com/39865976/181827674-02c0bdcc-85c2-43f7-9d96-ff1663689c82.png">

**Reply by sichen1234** [reply]
So how is the CEF(elec,BL,y) used?  What do they mean by "electricity displaced"?

As for additionality, IRR analysis is very sensitive to assumptions.  The old joke in finance is that IRR can be whatever you need to be.  

More importantly, for me I feel a credit needs to be additional during the vintage year when the activity happened.  This means that regardless of how it looked in 2008, is the carbon credit in fact additional in 2020?  Conversely, if the project shut down in 2020, would carbon credits be required to get it going again in that year?  

This depends on the status of landfill methane gas capture technology today.  Are they adopted on a large scale and common place?  Are they economically viable now?  Are they required by law (I've heard someone mention this though I'm not sure.)  Or are they discredited and no longer used for some reason (this also happens -- sometimes clean tech doesn't pan out)?  

This would require some research outside of the original project documentation and original certification, but I think it's important. 

This is also a good time to really reflect on what additionality truly means.

**Reply by brioux** [reply]
A couple thoughts, clarification based on my understanding of the project. 
Landfill gas credits are considered additional because the methane is being produced irrespective of the projects existence.

1. In absence of the project the methane is leaked and converted (destroyed) naturally in the atmosphere with a GWP of 29 to 80+ x CO2based on the timeline of interest.

Monitoring of the realized methane leakage is key. Comparing this against baseline (e.g.. 2006) is a good reference, but moving forward actual volumes captured and leaked in a given year would provided a much more accurate representation of  additionalilty. Volumes in 2006 may not reflect changes in landfill composition today - e.g. do to increased organic waste collection for more efficient management of methane through aerobic digestion. 

If the project captures and destroys 90% of total methane emissions this amount of “reduction” should be credited for the year. A combination of auditing captured methane volumes by the facility and satellite detection of total leakage from the landfill (GHGSat, and EDFs Methane SAT) could do the trick.

2. How the methane is destroyed should impact the credits value/price. If the captured methane is not sent to the energy conversion unit (simply flared) there is be a lost revenue potential, requiring a higher price on credits to make the project NPV positive.

3. The impact of discount rate on NPV is very interesting, and aligns with the work we are currently doing on sustainable financing of methane reduction projects for the oil and gas industry. If this methane reduction project can be NPV positive with a discount rate of 5%, one should consider the use of a Sustainability Linked Loan (SLL). This metric based instrument would award the project a 5% adjustable interest rate if it meets a targeted destruction rate, increased by increments for underperformance based on independent of assessment total captured+destroyed and leaked methane. The SSL could  make the project viable independent of fluctuations in the value/amount of carbon credits awarded. 

Using real/current performance metrics can also help remove perverse policy incentives, e.g., using outdated reference methane values for the landfill.

**Reply by sichen1234** [reply]
I agree with your point that a Sustainability Linked Loan is a better way to finance these kinds of projects.  Since this is sold as a carbon offset, though, we should evaluate it as one.  

A carbon offset transfers the emissions reduction claim to the buyer.  In that case, we have to make sure that it is truly additional, which means the project wouldn't have happened without the buyer payiing for the offset.  An analogy is a forest: The forest may be doing well, but if it's already a national park, then any offsetting claim is not additional.  Attempts to do so have been [received unfavorably](https://financialpost.com/pmn/business-pmn/u-s-public-forests-are-cashing-in-on-dubious-carbon-offsets).  

Here the key question is whether some form of landfill methane capture is commercially viable during the crediting year.  For example, [Waste Management is deploying landfill natural gas plants at scale](https://www.bizjournals.com/houston/news/2022/04/28/waste-management-rng-landfill-gas-investments.html)  If these kinds of projects, some financed by sustainability linked loans as you suggest, are viable, then the offset is not truly additional any more.

On the other hand, it doesn't have to reduce 90% of the methane -- it's just a claim for whatever amount of methane is destroyed.  I'm not sure then that a change in the landfill composition would change that.

**Reply by brioux** [reply]
Sorry, I misread. Credits are assigned based on the additional destruction of methane from 2010 to 2020 relative to the projected amount from the baseline year (2006)? Total vented is irrelevant, as long as more captured methane is destroyed. Makes sense
 
 if a new project is viable it can still be additional if it improves the baseline, with credits raising profitability and supporting future expansion plans - and more destruction.

 interested in how these credits are rolled forward.  When do new destruction levels transition to the next baseline? The first crediting period is June 4, 2010 to June 3, 2020. So no more credits after 2020, unless more destruction is achieved relative to a revised baseline projection?

Could credits be awarded using a shorter crediting period, e.g., every year based on the incremental destruction relative to a moving average, say the past 10 years?

And if annual destruction levels dip below the moving average, or whatever baseline projection is applied, how are credits adjusted? Penalties applied to the issuance of future credits?

What if a project consistently underperforms relative to the baseline, destroying less methane than the past? Should it be forced to buy back credits?

**Reply by hai-yr** [reply]
The project has been reviewed this year for renewal: 
<img width="678" alt="Screen Shot 2022-08-03 at 10 19 42 AM" src="https://user-images.githubusercontent.com/39865976/182670102-af3b0673-9591-4b18-b5ad-b06c2f24da9c.png">

Of note is that the credits we are looking at are 2020 credits, pre-renewal

**Reply by sichen1234** [reply]
Not reassessing additionality is a "hack".  Even if it is no mandated by law, it could be economically viable by now.

**Reply by sichen1234** [reply]
> if a new project is viable it can still be additional if it improves the baseline, with credits raising profitability and supporting future expansion plans - and more destruction.

Only the amount that is truly "additional," ie would not have happened without the credits, could be considered valid offsets.  So if the base project is viable for 100 tons but using credits allowed us to do 110 tons, then credits should only be issued for 10 tons.

It's always nice to expand climate projects, but on the other side, buyers are using these credits to fulfill their climate obligations.  If they can buy cheap, non-additional credits, then credits are really being used for greenwashing.  That is not good.
> 
> interested in how these credits are rolled forward. When do new destruction levels transition to the next baseline? The first crediting period is June 4, 2010 to June 3, 2020. So no more credits after 2020, unless more destruction is achieved relative to a revised baseline projection?
> 
> Could credits be awarded using a shorter crediting period, e.g., every year based on the incremental destruction relative to a moving average, say the past 10 years?

Credits should be issued based on the amount of methane destroyed in each year, based on audits for that year.
> 
> And if annual destruction levels dip below the moving average, or whatever baseline projection is applied, how are credits adjusted? Penalties applied to the issuance of future credits?

So in this case I don't think it should matter, because if the annual amount is lower in some years, then fewer credits should be issued in those years.  Though the audit reports should tell us how they came up with the amount of credits.
> 
> What if a project consistently underperforms relative to the baseline, destroying less methane than the past? Should it be forced to buy back credits?

I think that would cause fewer credits to be issued in those years.  In other words, the amount of credits issued each year should  be based on measured and audited methane destruction during that year.

**Reply by brioux** [reply]
@sichen1234  If the baseline set by a new project is not maintained wouldn’t the impact of the credits be cancelled out? It is not a matter of receiving less credits but undermining those already awarded.

Issuing credits over a longer horizon (say 10 years) resolves fluctuations in additionality. Would 10 years of additionality define an annual average or aggregation? the latter dilutes the real climate impact and value (in a rational market).

 I believe Issuing credits over a shorter 1 year horizon is more impactful. Also these credits can be rolled forwarded as long as the project maintains the new baseline set by the additionality.

I see credits as assets to cover discounted capital costs of investing in additionality. Project developers need a guarantee that the value linked to these assets will be accessible, but not necessarily available immediately. A project should operate at the new baseline beyond the start up/credit phase to preserve the right to sell them, or receive proceeds of those already sold.

**Reply by sichen1234** [reply]
The baseline and additionality of a voluntary offset are a little different than those for an instrument based purely on values of energy produced or methane captured.   This is even different than a compliance carbon credit, which is just based on the amount of measured emissions.

Baseline for an offset is the amount of emissions before the project, ie how much methane was coming out of the landfill without the gas capture project.

Additionality for an offset is whether the project would've happened without the carbon offset.  It is super important that a project is additional, or the buyer of the offset really shouldn't be able to claim the emissions reduction on their account. 

You're right that issuing credits over shorter, one year horizons is better.  It's important to reassess additionality and baseline more regularly.

It's not correct to use old methane CO2 equivalents.  It's really not good to waive the requirement for reassessing additionality ten years after a project started.  Ten years is probably too long to start with, and then being able to renew offsets after that is relying on assumptions that are very outdated.

**Reply by hai-yr** [reply]
Regarding CEF(elec,BL,y) in the larger equation: "No emission reductions are being claimed for grid-connected electricity generation that is offset by electricity produced from landfill gas destructed at the IES facility. Also, no thermal energy is produced using the landfill gas and the pre-project baseline year is 2006, so equation (1) is reduced to:
BEy = ( MDproject,y – MDBL,2006) * GWPCH4 "

Yes, in regards to the economics of the project, after 10 years it is really unclear what the profitability is. To me it would make sense if the project disclosed its return on investment so far and current economics of keeping the plant open... to show it still needs to sell credits to keep the project going.

**Reply by sichen1234** [reply]
So I wonder what "collects landfill gas and sends it to a landfill gas to energy facility for destruction" means?

What is "destruction" of landfill gas at a "landfill gas to energy facility"?  

I suppose we could find out, if the project were credible to start with.

**Reply by hai-yr** [reply]
![image](https://user-images.githubusercontent.com/39865976/183530971-30ca27c2-c720-4336-b5fc-9f947c6ee523.png)
from this article:
https://www.powermag.com/the-clean-and-dirty-of-landfill-gas-power/

**Reply by sichen1234** [reply]
That's what I thought landfill gas collection systems would do.  It generates some kind of energy.  So where did it go?  If it's not sent to the grid, did it get used at a county facility?  Was it to replace some grid energy, or for a new building or use that wasn't connected before?

This is not making sense to me.  Before the project, there already was a flare candle.  Was it not capturing all the methane, so that a lot more was being vented?  So now they are capturing the methane and burning it to make electricity, which reduces the emissions because burning the methane to CO2 is better than venting it directly.  That's great, but it doesn't mean the methane is "destructed."  It's just burnt, like flaring (righ @brioux?)  So they have to count the CO2 emissions of the methane being used for electricity.

**Comment by hai-yr** [comment]
Here is the written justification for the 10% discount rate:
<img width="842" alt="Screen Shot 2022-08-03 at 10 16 12 AM" src="https://user-images.githubusercontent.com/39865976/182669444-c9b31d72-0627-401f-a12e-ec02243e91f2.png">

**Reply by hai-yr** [reply]
seems like the "debt is unavailable" part is key for future landfill projects.

**Reply by sichen1234** [reply]
This does not sound right.  A municipality could borrow at much lower rates than 10% because their interest payments are tax exempt.  There are plenty of market-based bond rates that they could've used.

**Reply by hai-yr** [reply]
Hm I wonder why they claimed "debt is unavailable" then?

**Reply by sichen1234** [reply]
My guess is so that they could try to justify this much higher IRR.  If you go back to the original calculations, at a 5% discount rate the NPV is nearly $900,000.  The [long-term municipal bond rate](https://fred.stlouisfed.org/series/WSLB20) back in 2010 was about 4%.

**Reply by hai-yr** [reply]
From searching the project documents for some commentary on "debt": "The County has not issued General Obligations Bonds since 2001 except for refunding bonds. This is evidence of the County's very low tolerance for borrowing money to invest in discretionary projects and their reluctance to incur additional debt service expenses. Municipal governments have significant mandatory expenditures that are built into the tax rate. Political pressure may impact the municipalities' appetite for incurring more or less debt. A recent New York Times article detailed many of the most severe cases of municipal financial problems and was provided to the validator for review." - So it seems unclear/unlikely that they made an effort to try to secure debt for the project, but rather provided evidence of the difficulty/rareness of obtaining a loan.

**Reply by brioux** [reply]
@hai-yr your point makes a lot of sense… 

@sichen1234, the discount rate should refer to present value of cash raised for investment and the associated risk.

this is not the interest rate, which could reflec  efforts by governments or investors to minimize financial risk of project by linking rates to environmental performance. 

There is not a lot of history on the impact of sustainable financing for the risk of a project. So far these kinds of instruments have not had a significant impact on cost of capital. This needs to change

**Reply by hai-yr** [reply]
@brioux so do you think they had a reasonable expectation that they would not be able to secure debt, or that in not trying to go through the process of securing debt that they compromised the additionality of the credits they produced?

**Reply by brioux** [reply]
@hai-yr  their assumption is based on perceived risk of the project as its own business unit consistent with that of a private company. I guess this is the basis of the 10% rate taken from the referenced handbook.

Need to understand why they assume debt is unavailable when assessing their discount rate using a weighted average of cost of capital...

I don't think this compromised additionality. By applying a private sector discount rate they may have overestimated risk and expected return (value of credits) required by the project. However, I imagine this was a realistic assumption for the nature of the project. Sustainable financing initiatives should be considered, and related public policy initiatives, such as included in the Inflation Reduction Act mentioned by @NothingToContribute (Colin/agorafiliax)

**Reply by sichen1234** [reply]
Actually their line of reasoning goes against some fundamental theories of modern finance, such as the Capital Asset Pricing Model and Modigliani Miller.  What these theories say is that the required rate of return on an investment can be objectively determined from the market, not subjectively based on a particular investor (in this case a county government)'s preference for risk or using debt versus equity.

I don't think this project was additional back in 2006, and it wasn't even re-evaluated for additionality in 2020.

My conclusion here is that energy-related projects, including this one and the [Badhla solar project](https://github.com/opentaps/open-climate-investing/discussions/32), require too many economic assumptions about their additionality.  It's better, as @brioux suggests, to finance them with sustainability linked bonds and then try to get a premium for the energy output based on their renewable attributes.

**Comment by sichen1234** [comment]
I don't think this project was additional back in 2006, and it wasn't even re-evaluated for additionality in 2020, when the technology was much more prevalent.

Both this project and the [Badhla solar project](https://github.com/opentaps/open-climate-investing/discussions/32) relied on a lot of complex reasoning to justify their additionality.  They involved assumptions which were very specific to the projects but did not consider the overall market for the technologies.

I think a simpler and better test of additionality would be like what the Gold Standard adopted for renewable energy projects in 2021: Is this technology already 5% of the market?  If so then it cannot be considered additional any more.

**Comment by hai-yr** [comment]
My take is that there should definitely not have been a renewal of credits in 2020 without an assessment of the economics of running the project, and the amount of due diligence done on the possibility of securing debt in 2006 puts the original project's additionality into question.
